apps/src/CommandHistory.js

```diff
@@ -0,0 +1,86 @@
+/** @file Provide browseable command history to a textbox with limited depth. */
```

Comment:
Nit: deferring to Google

![image](https://cloud.githubusercontent.com/assets/206973/12864770/bc10078c-cc4b-11e5-8707-a2f0db983ee1.png)